.github/workflows/build.yml

```diff
@@ -8,14 +8,17 @@ on:
       - main
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
@rainerhahnekamp — I feel it's better to lock down to specific versions wherever possible, hence this change. But am happy to revert this bit if you feel otherwise.

Reply:
Nope, that's perfectly fine